app/drivers/tools/repair/java/SelfAPR.py

```diff
@@ -0,0 +1,122 @@
+import os
+from os.path import join
+
+from app.drivers.tools.repair.AbstractRepairTool import AbstractRepairTool
+
+
+class SelfAPR(AbstractRepairTool):
+    """
+    Requirements for this tool:
+    10 GB of VRAM, at most 7.0 CUDA (e.g. Nvidia V100) compute and 5 GB of RAM
```

Comment:
Does it really require CUDA 7.0? I ran it on a 4090 so it is looking good even on CUDA 12

Reply:
I am going to fix it for AlphaRepair as it also was working correctly

Reply:
I forgot to fix this requirement. I think I ran and installed the environment for Cuda 11.7.  I'm not sure for a higher version.

Reply:
Can just remove the CUDA requirement, it was mandatory for Recoder just to keep track of it

Reply:
done.